src/features/tooltip/components/PopupTooltip/PopupTooltip.tsx

```diff
@@ -0,0 +1,59 @@
+import { useAtom } from '@reatom/react';
+import { useCallback } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { Tooltip } from '@konturio/ui-kit';
+import { currentTooltipAtom } from '~core/shared_state/currentTooltip';
+import {
+  CSSTransitionWrapper,
+  fadeClassNames,
+} from '~components/CssTransitionWrapper/CssTransitionWrapper';
+import { parseLinksAsTags } from '~utils/markdown/parser';
+import { LinkRenderer } from '~components/LinkRenderer/LinkRenderer';
+import { closeOnLocationChangeAtom } from '../../atoms/closeOnLocationChangeAtom';
+import s from './PopupTooltip.module.css';
+
+export function PopupTooltip() {
+  const [tooltip, { resetCurrentTooltip }] = useAtom(currentTooltipAtom);
+  useAtom(closeOnLocationChangeAtom);
+
+  const closeHandler = useCallback(
+    (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      resetCurrentTooltip();
+      tooltip?.onClose?.(e, resetCurrentTooltip);
+    },
+    [resetCurrentTooltip, tooltip],
+  );
+
+  const outerClickHandler = useCallback(
+    (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+      tooltip?.onOuterClick?.(e, resetCurrentTooltip);
+    },
+    [resetCurrentTooltip, tooltip],
+  );
+
+  if (!tooltip) return null;
+
+  return (
+    <CSSTransitionWrapper in={Boolean(tooltip)} timeout={300} classNames={fadeClassNames}>
```

Comment:
fading animation stopped working locally

Reply:
found a way of how to fix the animation problem 
<img width="1792" alt="Screenshot 2023-01-13 at 11 17 04" src="https://user-images.githubusercontent.com/20676525/212271767-d2d37ff5-447c-4efa-b060-127289e3bd85.png">

but maybe there is a better way 🤷‍♂️

Reply:
cool, nice catch, thank you very much for investigation!